src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.html

```diff
@@ -90,12 +90,12 @@
         <li class="nav-item mr-2">
             <div class="btn-group-parent">
                 <button
-                    class="dataset-download-button starred rounded-left-2 border-1 border-right-0 btn-sm btn btn-group-item d-flex justify-content-center align-items-center"
+                    class="border-1 border btn-sm btn btn-group-item d-flex justify-content-center align-items-center"
```

Comment:
Please rename the "Download" button to "Get Data", and let's make it green - more visible.

Also, let's make the whole button clickable, not just its side arrow.

![image](https://github.com/kamu-data/kamu-web-ui/assets/204914/1bd6945c-d7e8-49ba-99fe-4a23893b9c4d)

Reply:
Fixed

![image](https://github.com/kamu-data/kamu-web-ui/assets/51016717/da1d2e55-a119-4418-b12d-42d4e7713b4d)